src/xrc/src/api/test.rs

```diff
@@ -867,3 +867,42 @@ fn get_exchange_rate_will_retrieve_rates_if_inflight_tracking_contains_any_symbo
         .expect("future should complete");
     assert!(matches!(result, Err(ExchangeRateError::Pending)));
 }
+
+/// This function tests that [get_exchange_rate] can retrieve crypto/USDT rates with one set of outbound
+/// calls.
+#[test]
+fn get_exchange_rate_can_retrieve_icp_usdt() {
+    let call_exchanges_impl = TestCallExchangesImpl::builder()
+        .with_get_cryptocurrency_usdt_rate_responses(hashmap! {
+            "ICP".to_string() => Ok(icp_queried_exchange_rate_mock())
+        })
+        .build();
+    let env = TestEnvironment::builder()
+        .with_cycles_available(XRC_REQUEST_CYCLES_COST)
+        .with_accepted_cycles(XRC_BASE_CYCLES_COST + XRC_OUTBOUND_HTTP_CALL_CYCLES_COST)
+        .build();
+    let request = GetExchangeRateRequest {
```

Comment:
What if the base and quote assets are reversed?
The same behavior should be observed in this case.

Reply:
Yes. Added a test to show this is the case.